libs/elements/cards/src/lib/elewa-news-card/elewa-news-card.component.html

```diff
@@ -0,0 +1,14 @@
+<div class="news-section">
+  <ul>
+    <h2 class="title">News</h2>
+    <li *ngFor="let news of newsData">
+      <div class="list-section">
+        <div class="content">
+          <p class="heading">{{news.title}}</p>
+          <p class="description">{{news.description}}</p>
+        </div>
+        <button class="read-more">{{news.buttonText}}</button>
+      </div>
```

Comment:
This should only render one news item. Work with one item only. Another component will render the entire section.